block1_bugrace_intro/README.md

```diff
@@ -2,7 +2,7 @@
 
 Nu ska du få lära känna Scratch och samtidigt skapa ett roligt spel som är enkelt även om du aldrig har jobbat med Scratch tidigare. Målet är att få en skalbagge att springa runt en bana och försöka undvika att hamna utanför banan genom att styra med vänster- och högerpil på tangentbordet.
 Ett exempel på det färdiga spelet finns på: [https://scratch.mit.edu/projects/27697024/](https://scratch.mit.edu/projects/27697024/)
-<a href="www.dn.se" target="_blank">test</a>
+<br><a href="http://www.dn.se" target="_blank">test</a>
```

Comment:
Hej,

Angående tabellen, det är ju egentligen fel att använda tabeller för att formatera innehåll, det var ett hack från början för matteboken för att få till en tvåspaltsdesign (som nu dessutom är en spalt på kodboken). Så krävs det dessutom att ni lägger in html i Markdown så blir det ju dubbelfel. Då tycker jag det är bättre att ha en heading 3 och text.

Palle




> On 3 dec. 2015, at 23:52, tinavincent <notifications@github.com> wrote:
> 
> In block1_bugrace_intro/README.md:
> 
> > @@ -2,7 +2,7 @@
> >  
> >  Nu ska du få lära känna Scratch och samtidigt skapa ett roligt spel som är enkelt även om du aldrig har jobbat med Scratch tidigare. Målet är att få en skalbagge att springa runt en bana och försöka undvika att hamna utanför banan genom att styra med vänster- och högerpil på tangentbordet.
> >  Ett exempel på det färdiga spelet finns på: [https://scratch.mit.edu/projects/27697024/](https://scratch.mit.edu/projects/27697024/)
> > -<a href="www.dn.se" target="_blank">test</a>
> > +<br><a href="http://www.dn.se" target="_blank">test</a>
> 
> Hej, html-länken var för att vi inte hittade något annat sätt att öppna länken i ett eget fönster mha markdown. Jag lyssnar gärna angående vad som gäller och ert tänk omkring detta. Det andra fallet av html handlar om en html-tabell. Den gjorde jag efter att ha frågat er, tror det var Tor, som föreslog att jag skulle kopiera ett html-avsnitt från Matteboken.se. Vi kanske kan höras om detta imorgon? Mvh /Tina Vincent
> …
> —
> Reply to this email directly or view it on GitHub.
>